pyproject.toml

```diff
@@ -17,7 +17,10 @@ keywords = [
   "install",
 ]
 authors = [
-    { name = "Philip Meier", email = "github.pmeier@posteo.de" },
+    { name = "Philip Meier" },
+]
+maintainers = [
+    { name = "Slicer Community"},
```

Comment:
LMK if you want to add a contact email here.

Reply:
It could be @jamesobutler and myself (@jcfr)